shotover-proxy/benches/redis_benches.rs

```diff
@@ -49,6 +50,9 @@ fn redis(c: &mut Criterion) {
     group.bench_with_input(
         "passthrough",
         || {
+            test_helpers::cert::generate_test_certs(Path::new(
+                "example-configs/redis-tls/tls_keys",
+            ));
```

Comment:
Why generate them in this bench? `redis-passthrough` doesn't need keys.

Reply:
good catch